src/pyramid/predicates.py

```diff
@@ -98,33 +98,43 @@ def __call__(self, context, request):
 
 class HeaderPredicate(object):
     def __init__(self, val, config):
-        name = val
-        v = None
-        if ':' in name:
-            name, val_str = name.split(':', 1)
-            try:
-                v = re.compile(val_str)
-            except re.error as why:
-                raise ConfigurationError(why.args[0])
-        if v is None:
-            self._text = 'header %s' % (name,)
-        else:
-            self._text = 'header %s=%s' % (name, val_str)
-        self.name = name
-        self.val = v
+        values = []
+
+        val = as_sorted_tuple(val)
+        for name in val:
+            v, val_str = None, None
+            if ':' in name:
+                name, val_str = name.split(':', 1)
+                try:
+                    v = re.compile(val_str)
+                except re.error as why:
+                    raise ConfigurationError(why.args[0])
+
+            values.append((name, v, val_str))
+
+        self.val = values
```

Comment:
I'm surprised that this was not covered by [Black](https://github.com/Pylons/pyramid/blob/master/HACKING.txt#L51). @mmerickel or @bertjwregeer should this be covered by Black?

Reply:
I don't understand the question. Why do you think it's not covered.

Reply:
`'` instead of `"`, or do we not care about that?

Reply:
This repository is specifically configured to not change from `'` to `"`. https://github.com/Pylons/pyramid/blob/master/pyproject.toml#L6